islandora_xacml_editor.module

```diff
@@ -324,8 +325,13 @@ function islandora_xacml_editor_manage_xacml_form(array $form, array &$form_stat
       ),
     );
   }
+  // Theme pager doesn't support url fragments in D7 so we insert manually.
```

Comment:
Copy-paste from https://github.com/Islandora/islandora_solution_pack_collection/commit/994c6df3d63609882bb9de9e7df1ab4aebb358d1#diff-2d69368f59ba81d5635cd8274dd66090R83 can we have this as a helper?
Thanks,
William Panting